### PR TITLE
UTDTA Talk - Nov 29

### PR DESCRIPTION
**Pull request by dasanchez**
New post for the events section of the website:
Talk on Nov 29 2017 for the UofT Decentralized Tech Association

### CODE REVIEW

File: _posts/2017-11-29-utdta-talk.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+Join us and the UofT Decentralized Tech Association (UTDTA) for a talk on decentralized tech and mesh networks!
+
+Guests:
+
+- Professor Peter Marbach
+- tomesh
+
+This event will be hosted by the UTDTA at {{page.startTime | date:"%-I:%M %P "}} in Room BA1180, [40 St. George Street]({{ page.locationLink }}).  
+
+All are welcome to join, just send an email to [{{ site.email }}](mailto:{{ site.email }}) so we know you are coming!
```
Comment (benhylau) [inline _posts/2017-11-29-utdta-talk.md]: Do we even need registration for this? UTDTA doesn't need it right?

**Review by benhylau — APPROVED** [review]

File: _posts/2017-11-29-utdta-talk.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title: tomesh @ UofT Decentralized Tech Association
```
Comment (dcwalk) [inline _posts/2017-11-29-utdta-talk.md]: We tend not to stylize "tomesh" in paragraphs, 
Maybe the title could be "Talk at U of T.... Meetup"

File: _posts/2017-11-29-utdta-talk.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+slug: decentralized tech talk
```
Comment (dcwalk) [inline _posts/2017-11-29-utdta-talk.md]: slug url stuff should have dashes between.

Comment (dcwalk) [reply _posts/2017-11-29-utdta-talk.md]: (and maybe match the title a little closer? i.e., add "talk" above or flip to "meetup".

File: _posts/2017-11-29-utdta-talk.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+text: The UTDTA is hosting tomesh for a talk on wireless mesh networks.
+location: Room BA1180, Bahen Centre for Information Technology, 40 St. George St  
+locationLink: http://osm.org/go/ZX6ByAqnV?
+date: 2017-11-29
+startTime: '18:00'
+endTime: '20:00'
+---
+
+Join us and the UofT Decentralized Tech Association (UTDTA) for a talk on decentralized tech and mesh networks!
```
Comment (dcwalk) [inline _posts/2017-11-29-utdta-talk.md]: I would maybe make clearer that we are invited guests, and not arranging?

File: _posts/2017-11-29-utdta-talk.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+Guests:
+
+- Professor Peter Marbach
+- tomesh
```
Comment (dcwalk) [inline _posts/2017-11-29-utdta-talk.md]: Again, would say "Toronto Mesh"

Comment (dcwalk) [reply _posts/2017-11-29-utdta-talk.md]: (and maybe our names 😉 )

File: _posts/2017-11-29-utdta-talk.md (hunk 1 above)
Comment (dcwalk) [reply _posts/2017-11-29-utdta-talk.md]: +1 that we should cut this

**Review by dcwalk — COMMENTED** [review]
Some thoughts on the phrasing

File: _posts/2017-11-29-utdta-talk.md (hunk 3 above)
Comment (benhylau) [reply _posts/2017-11-29-utdta-talk.md]: Good catch... should match the two exactly:

e.g.

```
title: "Install Party"
slug: install-party
```

**Review by benhylau — CHANGES_REQUESTED** [review]
See comments

**Comment by dcwalk** [comment]
I can update with these changes on break if that suits @dasanchez

**Comment by dasanchez** [comment]
@dcwalk yes please! I don't mind at all.

**Comment by dcwalk** [comment]
I'm going to merge this in as the adjustments have been made

**Pull request merged**